.github/workflows/pages.yml

```diff
@@ -0,0 +1,27 @@
+on: 
+  push:
+    branches: 
+    - master
+jobs:
+  run:
+    runs-on: ubuntu-18.04
+    steps:
+    # - uses: docker://pandoc/latex:2.9
+    - uses: actions/checkout@master
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.8' # Version range or exact version of a Python version to use, using semvers version range syntax.
+        architecture: 'x64' # (x64 or x86)
+    - run: |
+        sudo apt install pandoc
```

Comment:
Just a question, did the build not work without `pandoc`? I have it installed but it wasn't in my conda env, so if it's needed it was a very hidden dependency.

Reply:
i removed this line. we shouldnt need. i think it is necessary with latex.